mypyc/test-data/run-bools.test

```diff
@@ -228,3 +228,15 @@ def test_mix() -> None:
     print((y or 0) and True)
 [out]
 0
+
+[case testIsInstance]
+def test_built_in() -> None:
+    assert isinstance(True, bool)
```

Comment:
Optional nit: It's slightly better to use e.g. something like `b: Any = True` followed by `isinstance(b, bool)`, since a future version of mypyc might otherwise optimize the whole check away (here and elsewhere). It's not a big deal, but in the future it's often better to use expressions like these instead of `bool`/`int`/`str`/`bytes` literals in run tests:
* `int() + 1` instead of `1`
* `str() + "x"` instead of `"x"`

Container types are generally fine, since we are less likely to optimize them away. It's a loose convention that values with `Any` types and no-argument constructors for primitives such as `int()` are not optimized much.